modules/end-user-guide/examples/snip_che-configuring-a-workspace-to-use-an-intellij-based-ide.adoc

```diff
@@ -0,0 +1,39 @@
+.Prerequisites
+
+* Define the HTTPS resource defining the editor metadata: `__<URL_to_meta.yaml>__`. It can be:
+
+** Metadata for a prebuilt IntelliJ-based IDEs image:
++
+`++https://raw.githubusercontent.com/che-incubator/jetbrains-editor-images/meta/che-idea/latest.meta.yaml++`::
```

Comment:
I updated https://github.com/themr0c/che-docs/tree/test-intellij to use https://raw.githubusercontent.com/che-incubator/jetbrains-editor-images/meta/che-idea/latest.meta.yaml

I stiull have quota error when running https://workspaces.openshift.com/#https://github.com/themr0c/che-docs/tree/test-intellij

```
Add eth0 [10.129.2.58/23]

Pulling image "registry.redhat.io/codeready-workspaces/pluginbroker-metadata-rhel8@sha256:ca8ba32dd8c789cf4a384bf4bf1c63d845b370b7b1249dbbc0b7ebd1aab6437d"

Successfully pulled image "registry.redhat.io/codeready-workspaces/pluginbroker-metadata-rhel8@sha256:ca8ba32dd8c789cf4a384bf4bf1c63d845b370b7b1249dbbc0b7ebd1aab6437d" in 1.076910506s

Created container pluginbroker-metadata-rhel8-ca8ba32dd8

Started container pluginbroker-metadata-rhel8-ca8ba32dd8

Starting plugin metadata broker

List of plugins and editors to install

- errata-ai/vale-server/latest - The official Visual Studio Code extension for Vale and Vale Server.

- ms-vscode/vscode-github-pullrequest/latest - Pull Request and Issue Provider for GitHub

- joaompinto/asciidoctor-vscode/latest - Provides rich language support for AsciiDoc.

- timonwong/shellcheck/latest - Integrates ShellCheck into VS Code, a linter for Shell scripts.

- redhat/vscode-yaml/latest - YAML Language Support by Red Hat, with built-in Kubernetes syntax support

- redhat/vscode-xml/latest - XML Language Support by Red Hat

- vscode/typescript-language-features/latest - Provides rich language support for JavaScript and TypeScript.

- redhat/java/latest - Java Linting, Intellisense, formatting, refactoring, Maven/Gradle support and more...

- golang/go/latest - Rich Go language support for Visual Studio Code

- eclipse/che-idea/latest - IntelliJ IDEA Community running using Projector

All plugin metadata has been successfully processed

Failed to run the workspace: "Unrecoverable event occurred: 'FailedCreate', 'Error creating: pods "workspacex86l57t7tvrz6rzc.che-docs-78fdcb5b5c-q7rv4" is forbidden: exceeded quota: for-ffloreth-compute, requested: limits.memory=9402089472, used: limits.memory=0, limited: limits.memory=7Gi', 'workspacex86l57t7tvrz6rzc.che-docs-78fdcb5b5c'"
```

Reply:
Failed to start a Node.JS Express stack modified with:

```
  - type: cheEditor
    reference: 'https://raw.githubusercontent.com/che-incubator/jetbrains-editor-images/meta/che-idea/latest.meta.yaml'
```

```
Add eth0 [10.129.2.157/23]

Pulling image "registry.redhat.io/codeready-workspaces/pluginbroker-metadata-rhel8@sha256:ca8ba32dd8c789cf4a384bf4bf1c63d845b370b7b1249dbbc0b7ebd1aab6437d"

Successfully pulled image "registry.redhat.io/codeready-workspaces/pluginbroker-metadata-rhel8@sha256:ca8ba32dd8c789cf4a384bf4bf1c63d845b370b7b1249dbbc0b7ebd1aab6437d" in 1.235101749s

Created container pluginbroker-metadata-rhel8-ca8ba32dd8

Started container pluginbroker-metadata-rhel8-ca8ba32dd8

Starting plugin metadata broker

All plugin metadata has been successfully processed

List of plugins and editors to install

- vscode/typescript-language-features/latest - Provides rich language support for JavaScript and TypeScript.

- ms-vscode/node-debug2/latest - Node.js debugging support

- eclipse/che-idea/latest - IntelliJ IDEA Community running using Projector

AttachVolume.Attach succeeded for volume "pvc-060cb6cc-61ea-4111-8fb7-4be351bf37bf"

Add eth0 [10.129.7.231/23]

Pulling image "registry.redhat.io/codeready-workspaces/pluginbroker-artifacts-rhel8@sha256:0e6b054323fd8cfca3171fa5aaa25f48b1d4ebd098ca63e71d7f0a379e2a51a1"

Successfully pulled image "registry.redhat.io/codeready-workspaces/pluginbroker-artifacts-rhel8@sha256:0e6b054323fd8cfca3171fa5aaa25f48b1d4ebd098ca63e71d7f0a379e2a51a1" in 1.031687556s

Created container pluginbroker-artifacts-rhel8-0e6b054323

Started container pluginbroker-artifacts-rhel8-0e6b054323

Starting plugin artifacts broker

Cleaning /plugins dir

Processing plugin vscode/typescript-language-features/latest

Installing plugin extension 1/1

Downloading plugin from https://codeready-codeready-workspaces-operator.apps.sandbox-m2.ll9k.p1.openshiftapps.com/plugin-registry/v3/resources/github_com/redhat-developer/codeready-workspaces-vscode-extensions/releases/download/v2cebc1e/typescript-language-features-1.49.3.vsix

Processing plugin ms-vscode/node-debug2/latest

Installing plugin extension 1/2

Downloading plugin from https://codeready-codeready-workspaces-operator.apps.sandbox-m2.ll9k.p1.openshiftapps.com/plugin-registry/v3/resources/download_jboss_org/jbosstools/vscode/3rdparty/vscode-node-debug/node-debug-1.41.1.vsix

Installing plugin extension 2/2

Downloading plugin from https://codeready-codeready-workspaces-operator.apps.sandbox-m2.ll9k.p1.openshiftapps.com/plugin-registry/v3/resources/download_jboss_org/jbosstools/vscode/3rdparty/vscode-node-debug2/node-debug2-1.42.3.vsix

Saving log of installed plugins

All plugin artifacts have been successfully downloaded

Pulling image "registry.redhat.io/codeready-workspaces/jwtproxy-rhel8@sha256:c62c57ceb4b5f78eb62e43ee59e580e7df8cbe6e75a4e59ce344ea2c885c7129"

Successfully pulled image "registry.redhat.io/codeready-workspaces/jwtproxy-rhel8@sha256:c62c57ceb4b5f78eb62e43ee59e580e7df8cbe6e75a4e59ce344ea2c885c7129" in 1.066349803s

Created container che-jwtproxyitohl16v

Started container che-jwtproxyitohl16v

Pulling image "registry.redhat.io/codeready-workspaces/plugin-java8-rhel8@sha256:1f5ac84a66454ab4b7b31cb259d30d1a5675dd5a342c9c7dfc536846c8d9a9b1"

Successfully pulled image "registry.redhat.io/codeready-workspaces/plugin-java8-rhel8@sha256:1f5ac84a66454ab4b7b31cb259d30d1a5675dd5a342c9c7dfc536846c8d9a9b1" in 1.161224533s

Created container nodejs

Started container nodejs

Pulling image "registry.redhat.io/codeready-workspaces/plugin-java8-rhel8@sha256:1f5ac84a66454ab4b7b31cb259d30d1a5675dd5a342c9c7dfc536846c8d9a9b1"

Successfully pulled image "registry.redhat.io/codeready-workspaces/plugin-java8-rhel8@sha256:1f5ac84a66454ab4b7b31cb259d30d1a5675dd5a342c9c7dfc536846c8d9a9b1" in 993.209994ms

Created container 31d

Started container 31d

Pulling image "registry.redhat.io/codeready-workspaces/plugin-java8-rhel8@sha256:1f5ac84a66454ab4b7b31cb259d30d1a5675dd5a342c9c7dfc536846c8d9a9b1"

Successfully pulled image "registry.redhat.io/codeready-workspaces/plugin-java8-rhel8@sha256:1f5ac84a66454ab4b7b31cb259d30d1a5675dd5a342c9c7dfc536846c8d9a9b1" in 992.760423ms

Created container vscode-node-debug7ob

Started container vscode-node-debug7ob

Pulling image "quay.io/che-incubator/che-idea:latest"

Successfully pulled image "quay.io/che-incubator/che-idea:latest" in 45.097902872s

Created container che-ideaens

Started container che-ideaens

Pulling image "registry.redhat.io/codeready-workspaces/plugin-java8-rhel8@sha256:1f5ac84a66454ab4b7b31cb259d30d1a5675dd5a342c9c7dfc536846c8d9a9b1"

[che-ideaens] -> Prevent workspace from unexpected shutdown

[che-ideaens] -> Copy default configuration

[che-ideaens] -> Fri Sep 3 10:09:53 UTC 2021 - Perform request: 'curl -X PUT -H "Authorization: Bearer eyJhbGciOiJSUzI1NiIsImtpbmQiOiJtYWNoaW5lX3Rva2VuIiwia2lkIjoid29ya3NwYWNlY3ViazJqc3Y3N21lM3RrbiJ9.eyJ3c2lkIjoid29ya3NwYWNlY3ViazJqc3Y3N21lM3RrbiIsInVpZCI6IjgzNWFjZDEwLWI1ZjYtNGM3My1hM2FlLTQ5NzY1NTA2YzQ0YyIsImF1ZCI6IndvcmtzcGFjZWN1YmsyanN2NzdtZTN0a24iLCJuYmYiOi0xLCJ1bmFtZSI6ImZmbG9yZXRoIiwiaXNzIjoid3NtYXN0ZXIiLCJleHAiOjE2NjIxOTk2NzQsImlhdCI6MTYzMDY2MzY3NCwianRpIjoiY2Q4ZGE0NDctOTE3Yi00NzQxLWEyYjctY2Y5MzRiM2E1MWQwIn0.DJRva-AqFGNK0TskNH-HTfbX-JBYQXtlrjLrEnqO-J1BCo9Ga3lDmSXa-I2ie1u1aHdgDGoZILEXufvJ5gnWfFMjMQYLwC3WqHwHDnv3MguKW_SzsAbvPFe7q_o9eTQHHkr8byCxh37zgvONW-ka5b5N__s-Ekod4vfXF7P5Ld_QNuosxQS4CCNkv_gwSCLxHTlfvFp48c170AH2IGUYpkyNdZiP5LLCChic-P11lgAbGGWTpBVxVYLIocQiyga3SbqO8VnX30SWoURcZ2ZcbxwPyYk8-39bUpTTtle5_B1U5Fkky2meIkVBKzm0UumFhEUvC08ZzyQ9AhBphwrYVg" https://codeready-codeready-workspaces-operator.apps.sandbox-m2.ll9k.p1.openshiftapps.com/api/activity/workspacecubk2jsv77me3tkn'.

[che-ideaens] -> Found IDE: idea

[che-ideaens] -> % Total % Received % Xferd Average Speed Time Time Time Current

[che-ideaens] -> Dload Upload Total Spent Left Speed

[che-ideaens] ->

[che-ideaens] -> 0 0 0 0 0 0 0 0 --:--:-- --:--:-- --:--:-- 0

[che-ideaens] -> 0 0 0 0 0 0 0 0 --:--:-- --:--:-- --:--:-- 0

[che-ideaens] -> Fri Sep 3 10:09:53 UTC 2021 - Sleep for 1 minute.

[che-ideaens] -> OpenJDK 64-Bit Server VM warning: Option UseConcMarkSweepGC was deprecated in version 9.0 and will likely be removed in a future release.

[che-ideaens] -> [DEBUG] :: IdeState :: Starting attempts to attach IJ injector agent

[che-ideaens] -> [DEBUG] :: IdeState :: Starting attempts to initialize IDEA: fix AA and disable smooth scrolling (at start)

[che-ideaens] -> [DEBUG] :: IdeState :: Starting attempts to Getting IDE colors

[che-ideaens] -> [DEBUG] :: IdeState :: Starting attempts to search for editors

[che-ideaens] -> [DEBUG] :: ProjectorServer :: Daemon thread starts

[che-ideaens] -> [INFO] :: ProjectorServer :: ProjectorServer is starting on host 0.0.0.0/0.0.0.0 and port 8887

[che-ideaens] -> [INFO] :: HttpWsServerBuild

Failed to run the workspace: "The following containers have terminated: vscode-node-debug7ob: reason = 'Completed', exit code = 0, message = 'null' 31d: reason = 'Completed', exit code = 0, message = 'null'"
```

Reply:
Failure also with a golang stack:

```
Add eth0 [10.129.2.177/23]

Pulling image "registry.redhat.io/codeready-workspaces/pluginbroker-metadata-rhel8@sha256:ca8ba32dd8c789cf4a384bf4bf1c63d845b370b7b1249dbbc0b7ebd1aab6437d"

Successfully pulled image "registry.redhat.io/codeready-workspaces/pluginbroker-metadata-rhel8@sha256:ca8ba32dd8c789cf4a384bf4bf1c63d845b370b7b1249dbbc0b7ebd1aab6437d" in 1.12575765s

Created container pluginbroker-metadata-rhel8-ca8ba32dd8

Started container pluginbroker-metadata-rhel8-ca8ba32dd8

Starting plugin metadata broker

List of plugins and editors to install

- golang/go/latest - Rich Go language support for Visual Studio Code

- eclipse/che-idea/latest - IntelliJ IDEA Community running using Projector

All plugin metadata has been successfully processed

Successfully assigned ffloreth-dev/workspacew225vh6rny971zb5.go-cli-66c4474fd6-bkms4 to ip-10-0-142-184.us-east-2.compute.internal

Multi-Attach error for volume "pvc-060cb6cc-61ea-4111-8fb7-4be351bf37bf" Volume is already exclusively attached to one node and can't be attached to another

AttachVolume.Attach succeeded for volume "pvc-060cb6cc-61ea-4111-8fb7-4be351bf37bf"

Add eth0 [10.129.7.232/23]

Pulling image "registry.redhat.io/codeready-workspaces/pluginbroker-artifacts-rhel8@sha256:0e6b054323fd8cfca3171fa5aaa25f48b1d4ebd098ca63e71d7f0a379e2a51a1"

Successfully pulled image "registry.redhat.io/codeready-workspaces/pluginbroker-artifacts-rhel8@sha256:0e6b054323fd8cfca3171fa5aaa25f48b1d4ebd098ca63e71d7f0a379e2a51a1" in 1.116940952s

Created container pluginbroker-artifacts-rhel8-0e6b054323

Started container pluginbroker-artifacts-rhel8-0e6b054323

Starting plugin artifacts broker

Processing plugin golang/go/latest

Installing plugin extension 1/1

Plugin already downloaded

Saving log of installed plugins

All plugin artifacts have been successfully downloaded

Pulling image "registry.redhat.io/codeready-workspaces/jwtproxy-rhel8@sha256:c62c57ceb4b5f78eb62e43ee59e580e7df8cbe6e75a4e59ce344ea2c885c7129"

Successfully pulled image "registry.redhat.io/codeready-workspaces/jwtproxy-rhel8@sha256:c62c57ceb4b5f78eb62e43ee59e580e7df8cbe6e75a4e59ce344ea2c885c7129" in 1.389624184s

Created container che-jwtproxy566faft4

Started container che-jwtproxy566faft4

Pulling image "registry.redhat.io/codeready-workspaces/stacks-golang-rhel8@sha256:a4c848642e5cbd2e503915932c27342c94904763caa9085a866cb39c7e443335"

Successfully pulled image "registry.redhat.io/codeready-workspaces/stacks-golang-rhel8@sha256:a4c848642e5cbd2e503915932c27342c94904763caa9085a866cb39c7e443335" in 1.016881913s

Created container go-cli

Started container go-cli

Pulling image "registry.redhat.io/codeready-workspaces/stacks-golang-rhel8@sha256:a4c848642e5cbd2e503915932c27342c94904763caa9085a866cb39c7e443335"

Successfully pulled image "registry.redhat.io/codeready-workspaces/stacks-golang-rhel8@sha256:a4c848642e5cbd2e503915932c27342c94904763caa9085a866cb39c7e443335" in 1.01082307s

Created container vscode-go8q6

Started container vscode-go8q6

Pulling image "quay.io/che-incubator/che-idea:latest"

Successfully pulled image "quay.io/che-incubator/che-idea:latest" in 244.123721ms

Created container che-ideadw5

Started container che-ideadw5

Pulling image "registry.redhat.io/codeready-workspaces/stacks-golang-rhel8@sha256:a4c848642e5cbd2e503915932c27342c94904763caa9085a866cb39c7e443335"

[che-jwtproxy566faft4] -> time="2021-09-03T10:18:25Z" level=info msg="Starting reverse proxy (Listening on ':4400')"

Failed to run the workspace: "The following containers have terminated: vscode-go8q6: reason = 'Completed', exit code = 0, message = 'null'"
```